INSTALL.md

```diff
@@ -1,115 +1,226 @@
+
 
 Install the non-python dependencies
 -----------------------------------
 
 * [PostgreSQL](http://www.postgresql.org/download/) (minimum required version 9.3.0, we have tested most extensively with 9.3)
 * [redis-server](http://redis.io) (we have tested most extensively with 2.8.17)
+* [hdf5](https://www.hdfgroup.org/HDF5/)
+
+There are several options to install these dependencies depending on your needs:
+
+- You could install them via conda, however, the anaconda repository may not have the exact versions of these that you want. 
+- You could install the exact versions we tested as per instructions on their websites, and thus make their available globally on your operating system. This may or may not cause problems if other software on your system requires a differing, specific version of the packages. These instructions were tested with this method.
+- You could setup a full development environment with [Vagrant](https://www.vagrantup.com/), and continue using conda under it to primarily manage python dependencies.
+
+### PostgreSQL installation on Mac OS X
 
-Install both of these packages according to the instructions on their websites.
-You'll then need to ensure that the postgres binaries (for example, ``psql``)
-are in your executable search path (``$PATH`` environment variable). For
-example if you are using Postgres.app on OS X, you can do this by addint
-the following line to your `.bash_profile`:
+For Mac OS X, you can either install postgres through the [Postgres.app](https://www.postgresql.org/download/macosx/) or [homebrew](http://brew.sh). These instructions were tested with the Postgres.app v9.3
+
+You'll then need to ensure that the postgres binaries (for example, ``psql``) are in your executable search path (``$PATH`` environment variable). If you are using Postgres.app on OS X, you can do this by running the following, though you may have to replace`~/.bash_profile`with `~/.zshrc` if you're using zshell rather than the built-in bash, and you may have to change the version number `Versions/9.3/` to the exact one that you are installing:
 
 ```bash
-export PATH=$PATH:/Applications/Postgres.app/Contents/Versions/9.3/bin/
+echo 'export PATH="$PATH:/Applications/Postgres.app/Contents/Versions/9.3/bin/"' >> ~/.bash_profile
 ```
 
-Install Qiita and its python dependencies
------------------------------------------
+### Redis-server installation on Mac OS X
 
-Then, you can use pip to install Qiita, which will also install its python dependencies.
+You can install redis-server v2.8.x using homebrew as follows:
 
 ```bash
-pip install numpy
-pip install https://github.com/biocore/mustached-octo-ironman/archive/master.zip --no-deps
-pip install qiita-spots
+brew update
+brew install homebrew/versions/redis28
 ```
 
-Install Qiita development version and its python dependencies
--------------------------------------------------------------
+### hdf5 installation on Mac OS X
```

Comment:
It can be installed with conda, so you can add it to the installations once conda is installed.